subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1781,8 +1781,12 @@ int bt_mesh_light_ctrl_srv_disable(struct bt_mesh_light_ctrl_srv *srv)
 	atomic_clear_bit(&srv->flags, FLAG_CTRL_SRV_MANUALLY_ENABLED);
 
 	if (!is_enabled(srv)) {
-		/* Restart resume timer even if the server has already been disabled: */
-		schedule_resume_timer(srv);
+		if (atomic_test_bit(&srv->flags, FLAG_RESUME_TIMER)) {
+			/* Restart resume timer even
+			 * if the server has already been disabled:
+			 */
+			schedule_resume_timer(srv);
```

Comment:
At line 1794, schedule_resume_timer() is called again even if the FLAG_RESUME_TIMER is not set.
Should be added this if statement to that line too?

Reply:
schedule_resume_timer() is also called inside scene_recall(). 
Not sure if the scene_recall can be started even if LC was not run before.

Reply:
Hm. Seems this is the specification requirement, isn't this?

```
5.1.3.1.2 Scene Recall operation
...
The Scene Recall operation shall recall all values for all states specified as Stored with Scene for all models present on elements with addresses starting with the address of the element on which the Scene Server is present and ending either with the address preceding the address of the element on which the next Scene Server is present or with the address of the last element, whichever comes first, except for the states that have been explicitly excluded from the operation.
```

Reply:
> Hm. Seems this is the specification requirement, isn't this?
> 
> ```
> 5.1.3.1.2 Scene Recall operation
> ...
> The Scene Recall operation shall recall all values for all states specified as Stored with Scene for all models present
>  on elements with addresses starting with the address of the element on which the Scene Server is present and 
> ending either with the address preceding the address of the element on which the next Scene Server is present or 
> with the address of the last element, whichever comes first, except for the states that have been explicitly excluded 
> from the operation.
> ```
Yes, from spec Scene Recall shall be called even if LC was not run before.
But for resume timer (inside scene_recall() function), we should add the same if statement to only schedule resume timer if LC was run before, otherwise we will Enable Light Control during a scene recall.

Reply:
@asousa-trd, please, take a look at the changes. Cannot rerequest your review for some reason.

Reply:
Hi @alxelax , LGTM :thumbsup: